File: docs/docs/api-reference/iam/api_user/v1/examples/go/activate-api-user.go
package main

import (
	"context"
	"log"

	api_userv1 "github.com/meshtrade/api/go/iam/api_user/v1"
)

func main() {
	// Create client (see client-setup.go for details)
	client, err := api_userv1.NewApiUserServiceGRPCClient(
		api_userv1.WithAddress("localhost", 8080),
		api_userv1.WithTLS(false),
		api_userv1.WithAPIKey("your-api-key"),
		api_userv1.WithGroup("your-group-id"),
	)
	if err != nil {
		log.Fatalf("Failed to create client: %v", err)
	}
	defer client.Close()

	// Activate the API user by name
	apiUser, err := client.ActivateApiUser(
		context.Background(),
		&api_userv1.ActivateApiUserRequest{
			Name: "api_users/01HPQR2S3T4U5V6W7X8Y9Z0123", // Replace with actual API user name
		},
	)
	if err != nil {
		log.Fatalf("Failed to activate API user: %v", err)
	}

	log.Printf("Activated API user: %s", apiUser.Name)
	log.Printf("State: %v", apiUser.State)
}

File: docs/docs/api-reference/iam/api_user/v1/examples/python/activate-api-user.py
from meshtrade.iam.api_user.v1 import (
    ApiUserServiceGRPCClient,
    ClientOptions,
    ActivateApiUserRequest,
)


def main():
    # Create client (loads credentials from MESH_API_CREDENTIALS)
    options = ClientOptions()
    client = ApiUserServiceGRPCClient(options)
    
    with client:
        # Create request
        request = ActivateApiUserRequest(
            name="api_users/01HPQR2S3T4U5V6W7X8Y9Z0123"  # Replace with actual API user name
        )
        
        # Make the API call
        response = client.activate_api_user(request)
        
        print(f"Activated API user: {response.name}")
        print(f"State: {response.state}")


if __name__ == "__main__":
    main()
